File: openapi/code_samples/PHP/authorization/recover.php
<?php
$client = new Client();
$headers = [
  'Authorization' => '',
  'Content-Type' => 'application/json'
];
$body = '{
  "method": "email",
  "wordToMatch": "m_yoni3",
  "typeUser": "manager"
}';
$request = new Request('POST', 'https://api.tizo.co/api/v1/auth/recovery-password/', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();

File: openapi/code_samples/csharp/authorization/recover.cs
var client = new RestClient("https://staging-api.tizo.co/api/v1/auth/recovery-password/");
client.Timeout = -1;
var request = new RestRequest(Method.POST);
request.AddHeader("Authorization", "");
request.AddHeader("Content-Type", "application/json");
var body = @"{
" + "\n" +
@"  ""method"": ""email"",
" + "\n" +
@"  ""wordToMatch"": ""m_yoni3"",
" + "\n" +
@"  ""typeUser"": ""manager""
" + "\n" +
@"}";
request.AddParameter("application/json", body,  ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);